### ISSUE TITLE
[Task]: Update issue templates with new hierarchy and labels

### ISSUE THREAD

**Issue by widal001** [HHS/simpler-grants-gov#426]
### Summary

Now that the group has settled on the following issue hierarchy: `milestone > epic > issue` we need to update our existing issue templates to reflect this new hierarchy and update the default labels.

### Dependencies

- [x] #320 

### Specification

Update the list of issue templates to reflect this list

1. Bug Report
    - Sections: No change
    - Labels: `bug`
2. Feature Request
    - Sections: No change
    - Labels: `enhancement`
3. Internal - 30k
    - Sections
      - Description
      - Business goals
      - Dependencies
      - Functionality
      - Definition of Done
    - Labels: `deliverable: 30k ft`
4. Internal - 10k
    - Sections
      - Definition
      - Business goals
      - In scope
      - Out of scope
      - Tasks
      - Definition of Done
    - Labels: `deliverable: 10k ft`
5. Internal - Task
    - Sections: No Change
    - Labels: 
6. Internal - ADR
    - Sections: No Change
    - Labels: `docs: adr`
7. Internal - Deliverable spec
    - Sections: No Change
    - Labels: `docs: deliverable`

### Acceptance criteria

- [x] The issue templates have been updated to reflect the requirements listed above
- [x] The project automations use the new labels to automatically add tickets to a project

**Comment by widal001** [comment]
Reach out to the engineering team to see if they can support

**Comment by sarahknoppA6** [comment]
Waiting on the project convention updates in GitBook.
